igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/client/JenkinsClient.groovy

```diff
@@ -41,7 +41,7 @@ interface JenkinsClient {
     @GET('/api/xml?tree=jobs[name,jobs[name,jobs[name,jobs[name,jobs[name,jobs[name,jobs[name,jobs[name,jobs[name,jobs[name]]]]]]]]]]')
     JobList getJobs()
 
-    @GET('/job/{jobName}/api/xml?exclude=/*/build/action[not(totalCount)]&tree=builds[number,url,duration,timestamp,result,building,url,fullDisplayName,actions[failCount,skipCount,totalCount]]')
+    @GET('/job/{jobName}/api/xml?exclude=/*/build/action[not(totalCount)]&tree=builds[number,url,duration,timestamp,result,building,url,fullDisplayName,artifacts[displayPath,fileName,relativePath],actions[failCount,skipCount,totalCount]]')
```

Comment:
Why exclude these? Aren't these fields what the artifact reference is ultimately derived from?

Reply:
@eleftherias and I noticed that if we didn't add that as part of the exclude we wouldn't get the details to derive the reference